test/functions/activation/leaky_relu_test.rb

```diff
@@ -3,13 +3,11 @@
 require 'chainer/functions/activation/leaky_relu'
 
 class Chainer::Functions::Activation::LeakyReLUTest < Test::Unit::TestCase
-  data = {
-    'test1' => {shape: [3, 2], dtype: xm::SFloat},
-    'test2' => {shape: [], dtype: xm::SFloat},
-    'test3' => {shape: [3, 2], dtype: xm::DFloat},
-    'test4' => {shape: [], dtype: xm::DFloat}}
 
-  def _setup(data)
+  data(:shape, [[3, 2], []],             keep: true)
+  data(:dtype, [xm::SFloat, xm::DFloat], keep: true)
+
+  def setup()
```

Comment:
Do you need ()?
It looks not rubyish.

Reply:
ditto for others

Reply:
> Do you need ()?

Does this point to **setup()** -> **setup**?
Or does this point to **data(:shape, [[3, 2], []],             keep: true)** -> **data :shape, [[3, 2], []],             keep: true**?

Reply:
Of course, 1. All def lines